packages/office-ui-fabric-react/src/components/DetailsList/DetailsRow.tsx

```diff
@@ -181,14 +185,16 @@ export class DetailsRow extends BaseComponent<IDetailsRowProps, IDetailsRowState
     const isContentUnselectable = selectionMode === SelectionMode.multiple;
 
     return (
-      <div
+      <FocusZone
+        isInnerZoneKeystroke={ isEnter }
+        direction={ FocusZoneDirection.horizontal }
         ref='root'
```

Comment:
might as well change this one to `componentRef` too. I'm surprised it is still there. I thought I had scrubbed ref=